src/components/Breadcrumbs/BreadcrumbItem.tsx

```diff
@@ -45,7 +45,7 @@ const BreadcrumbItem: React.FC<BreadcrumbItemProps> = ({
         <BreadcrumbLink
           color="secondary"
           href={href}
-          iconName={iconName}
+          iconStart={{ iconName }}
```

Comment:
Please fix the `hasIcon` condition

Reply:
No need to change public API here. The issue was that the Breadcrumb link with icon contained also the label, which was regression because label should be hidden in this case

Reply:
🤦‍♀️  I get it